org.eclipse.scanning.event.ui/src/org/eclipse/scanning/event/ui/view/StatusQueueView.java

```diff
@@ -680,7 +632,7 @@ protected void openSelection() {
 		final StatusBean [] beans = getSelection();
 		if (beans.length == 0) {
 			MessageDialog.openInformation(getViewSite().getShell(), "Please select a run", "Please select a run to open.");
-            return;
+			return;
 		}
 
 		// TODO FIXME Change to IScanBuilderService not selections so that it works with e4.
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1134)
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)